libsolidity/codegen/ExpressionCompiler.cpp

```diff
@@ -1504,12 +1510,9 @@ void ExpressionCompiler::appendArithmeticOperatorCode(Token::Value _operator, Ty
 		}
 		else
 		{
+			solUnimplemented("Not yet implemented - FixedPointType exponentiation");
 			std::string code = R"(
 			{
-				if lt(exponent, 0) {
-					invalid()
-				}
-
 				let factor := <factor>
 				if iszero(eq(exponent, mul(div(exponent, factor), factor))) { // Check if exponent has no decimal value
```

Comment:
s/decimal/integer/

Reply:
I'm sorry, what?

Reply:
I think you should rather say `check if exponent has integer value` or `non-fractional` - the word `decimal` is too ambigous in this context: `10` is a decimal number (because it is written using the decimal system), but it is not fractional.